app/javascript/src/components/Dashboard/Notes/List/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+
+import Note from "./Note";
+
+import { SAMPLE_NOTES } from "../constants";
+import EditNotePane from "../Pane/EditNote";
```

Comment:
Let's make it a practice not to prefix or suffix component/file/folder names with `Pane, Modal, Form, Page etc`. Here, `Edit.jsx`  would be correct since `Edit.jsx` resides inside of `Notes`. 
If you want, you could split it as follows - 
1. `Notes/Edit/index.jsx` contains the `Pane` related logic.
2. `Notes/Edit/Form.jsx` contains the form related logic.